com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPRequest.java

```diff
@@ -100,7 +109,14 @@ public void setParams(String value) throws Exception {
 	 * @throws Exception
 	 */
 	public HTTPResponse sendRequest(IAuthenticate auth) throws Exception {
-		HttpClient client = new DefaultHttpClient();
+		HttpClient client;
+		if(insecure) {
```

Comment:
Minor point, but since you defined a getter method (isInsecure()) best practice is to use it.